webrender/src/tiling.rs

```diff
@@ -1723,6 +1794,26 @@ impl ScreenTile {
                             alpha_task_stack.push(prev_task);
                         }
                     }
+
+                    // Create a task for the layer mask, if needed
```

Comment:
Does this do renders for axis aligned regions, that can be handled without a mask?